File: pueblo_golf_tournament_mobile/lib/api/responses/registration/registered-team-images-dto.dart
import 'package:json_annotation/json_annotation.dart';
import 'package:pueblo_golf_tournament_mobile/api/responses/base-response.dart';

part 'registered-team-images-dto.g.dart';

@JsonSerializable()
class RegisteredTeamImagesDto extends BaseResponse {
  final String paymentImageUrl;
  final String logoImageUrl;
  RegisteredTeamImagesDto({
    required super.message,
    required this.paymentImageUrl,
    required this.logoImageUrl,
  });

  factory RegisteredTeamImagesDto.fromJson(Map<String, dynamic> json) =>
      _$RegisteredTeamImagesDtoFromJson(json);

  Map<String, dynamic> toJson() => _$RegisteredTeamImagesDtoToJson(this);
}

File: pueblo-golf-tournament-api/Dto/Outgoing/RegisteredTeamImagesDto.cs
using pueblo_golf_tournament_api.Base;

namespace pueblo_golf_tournament_api.Dto.Outgoing
{
    public class RegisteredTeamImagesDto : BaseResponse
    {
        public String? PaymentImageUrl { get; set; }
        public String? LogoImageUrl { get; set; }
    }
}